build.cmd

```diff
@@ -1,3 +1,4 @@
 @ECHO OFF
 SETLOCAL
-PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; try { & '%~dp0build.ps1' %*; exit $LASTEXITCODE } catch { write-host $_; exit 1 }"
+PowerShell -NoProfile -NoLogo -ExecutionPolicy Bypass -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = ''; try { & '%~dp0build.ps1' %*; exit $LASTEXITCODE } catch { write-host $_; exit 1 }"
+ECHO build.cmd completed
```

Comment:
@JunTaoLuo I just had a look at the build for this PR and notice the Windows x64 build step (https://dev.azure.com/dnceng/public/_build/results?buildId=224899&view=logs&j=f31c9f97-4411-58e7-49ac-fc73f645e6b6) should have failed but did not. This line loses the exit code from the PowerShell invocation.

And, jobs are still timing out even though your change is in. For example the internal build https://dnceng.visualstudio.com/internal/_build/results?buildId=224691&view=logs&j=f31c9f97-4411-58e7-49ac-fc73f645e6b6 shows:
```
2019-06-14T02:35:06.4970654Z     84 Warning(s)
2019-06-14T02:35:06.4971845Z     16 Error(s)
2019-06-14T02:35:06.4972524Z
2019-06-14T02:35:06.4974642Z Time Elapsed 00:39:21.51
2019-06-14T04:52:55.8328920Z ##[error]The operation was canceled.
2019-06-14T04:52:55.8691848Z ##[section]Finishing: Build x64
```

The above looks very much like MSBuild itself is not exiting after it wraps up, not PowerShell.

Let me know if you can't tackle this almost immediately.

Reply:
Let me remove the logging commands and investigate

Reply:
I don't have access to the diagnostics logs since they were not uploaded in time after the timeout grace period. I'm going to move that build step earlier and see if it helps.

Reply:
Turns out it's not really configurable. Artifacts are uploaded after test results are uploaded. Hmm.... this mechanism might not work.